lib/nimbusRecipe.ts

```diff
@@ -250,4 +253,25 @@ null,
 
     return `https://experimenter.services.mozilla.com/nimbus/${encodeURIComponent(this._rawRecipe.slug)}/summary#${screenshotsAnchorId}`
   }
+
+  /**
+   * @returns true if this recipe is a messaging recipe.
+   */
+  isMsgRecipe() {
+    return this.usesMessagingFeatures()
+  }
```

Comment:
I noticed that `isMsgRecipe()` is essentially just using `useMessagingFeatures()`. I didn't add unit tests because of this but wanted to ask if this function should then just be removed or if there might be future additions to this function which is why we're keeping it?

Reply:
Good call, we can just remove `isMsgRecipe()`.